src/conf_mode/protocols_mld.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020-2023 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from ipaddress import IPv6Address
+from sys import exit
+
+from vyos import ConfigError
+from vyos.config import Config
+from vyos.utils.process import process_named_running
+from vyos.utils.process import call
+from vyos.template import render
+from signal import SIGTERM
+
+from vyos import airbag
+airbag.enable()
+
+# Required to use the full path to pim6d, in another case daemon will not be started
+pimd_cmd = f'/usr/lib/frr/pim6d -d -F traditional --daemon -A 127.0.0.1'
+
+config_file = r'/tmp/mld.frr'
+
+def get_config(config=None):
+    if config:
+        conf = config
+    else:
+        conf = Config()
+    mld_conf = {
+        'mld_conf' : False,
+        'old_ifaces'    : {},
+        'ifaces'    : {}
+    }
+    if not (conf.exists('protocols mld') or conf.exists_effective('protocols mld')):
+        return None
```

Comment:
The config exists and effective are old implementation of configuration. 
You should use get_config_dict ad in this example https://github.com/vyos/vyos-1x/blob/d3edda22573fb9c9d1c469f14f5a3eec9ca512a5/src/conf_mode/protocols_babel.py#L41

Reply:
done